helics/capi.py

```diff
@@ -828,8 +853,8 @@ def __repr__(self):
             class_name=self.__class__.__name__, identifier=identifier, address=address, id=hex(id(self)),
         )
 
-    def __del__(self):
-        helicsCoreFree(self)
+    # def __del__(self):
```

Comment:
I'm personally a fan of just deleting lines like this instead of commenting them out :) The Git history can be consulted to find old code. To each their own, however! 😄

Reply:
Resolved in https://github.com/GMLC-TDC/pyhelics/pull/26/commits/c226212d2d6cac119f85d477ce360e626e8ccbd3.